forge-gui/res/cardsfolder/upcoming/mizzix_replica_rider.txt

```diff
@@ -0,0 +1,15 @@
+Name:Mizzix, Replica Rider
+ManaCost:4 R
+Types:Legendary Creature Goblin Wizard
+PT:4/5
+K:Flying
+T:Mode$ SpellCast | ValidCard$ Card.wasNotCastFromYourHand | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigCopy | TriggerDescription$ Whenever you cast a spell from anywhere other than your hand, you may pay {1}{U/R}. If you do, copy that spell and you may choose new targets for the copy. If the copy is a permanent spell, it gains haste and "At the beginning of the end step, sacrifice this permanent."
```

Comment:
Please add for card detail window: (A copy of a permanent spell becomes a token.)